ams/server/src/main/java/com/netease/arctic/server/catalog/ExternalCatalog.java

```diff
@@ -137,6 +168,44 @@ public AmoroTable<?> loadTable(String database, String tableName) {
     return doAs(() -> unifiedCatalog.loadTable(database, tableName));
   }
 
+  protected void validateTableIdentifier(TableIdentifier tableIdentifier) {
+    if (!name().equals(tableIdentifier.getCatalog())) {
+      throw new IllegalMetadataException("Catalog name is error in table identifier");
+    }
+  }
+
+  private String getDatabaseDesc(String database) {
+    return name() + '.' + database;
+  }
+
+  protected void increaseDatabaseTableCount(String databaseName) {
+    // do not handle database operations
+  }
+
+  @Override
+  public TableMetadata createTable(TableMetadata tableMetadata) {
+    // catalog which support mixed hive/mixed iceberg format can createTable.
+    validateTableIdentifier(tableMetadata.getTableIdentifier().getIdentifier());
+    ServerTableIdentifier tableIdentifier = tableMetadata.getTableIdentifier();
+    doAsTransaction(
+        () -> doAs(TableMetaMapper.class, mapper -> mapper.insertTable(tableIdentifier)),
+        () -> doAs(TableMetaMapper.class, mapper -> mapper.insertTableMeta(tableMetadata)),
+        () ->
+            doAsExisted(
+                CatalogMetaMapper.class,
+                mapper -> mapper.incTableCount(1, name()),
+                () -> new ObjectNotExistsException(name())),
+        () -> increaseDatabaseTableCount(tableIdentifier.getDatabase()));
+
+    return getAs(
+        TableMetaMapper.class,
+        mapper ->
+            mapper.selectTableMetaByName(
+                tableIdentifier.getCatalog(),
+                tableIdentifier.getDatabase(),
+                tableIdentifier.getTableName()));
+  }
+
```

Comment:
This should be a temporary solution. 
The creation of the MixedHive format should not communicate with AMS in the future, and the metadata of the MixedHive format should not be stored in AMS in the future